lemur/plugins/lemur_email/plugin.py

```diff
@@ -55,21 +55,25 @@ def send_via_smtp(subject, body, targets):
 
 def send_via_ses(subject, body, targets):
     """
-    Attempts to deliver email notification via SMTP.
+    Attempts to deliver email notification via SES service.
     :param subject:
     :param body:
     :param targets:
     :return:
     """
     client = boto3.client("ses", region_name="us-east-1")
```

Comment:
curious, if this should be a config: 
```
region_name="us-east-1"
```

Reply:
I think it absolutely should, though I noticed it's a widespread problem throughout all the AWS code. I think for now it might be best to leave it hardcoded and consider reworking the whole code base to take region as a configuration parameter.

Reply:
Actually, I think I misread something. At a second glance, most regions come from plugin configs and aren't hardcoded. I'll fix it here now.